front/src/common/redux/groups/groups.action.js

```diff
@@ -1,8 +1,15 @@
-import { SET_CURRENT_GROUP } from './types';
+import { SET_CURRENT_GROUP, CREATE_GROUP } from './types';
 
 export const setCurrentGroup = (data) => {
   return {
     type: SET_CURRENT_GROUP,
     data
   }
 };
+
+export const createGroup = (data) => {
+  return {
```

Comment:
Please, use the simplified syntax `= ({ ... })` instead of return statement

Reply:
done